pkg/file/file_test.go

```diff
@@ -0,0 +1,41 @@
+package file
+
+import (
+	"github.com/go-git/go-billy/v5"
+	"github.com/go-git/go-billy/v5/helper/polyfill"
+	"github.com/go-git/go-billy/v5/test"
+	"github.com/go-git/go-git/v5"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+//go:generate mockgen -source=file_test.go -destination=mocks/mock_file.go -package=mocks FileMockInterface
+type FileMockInterface interface {
+	billy.File
+}
+
+func TestOpen(t *testing.T) {
+
+	ruleFile := RuleFile{
+		Path: "/",
+	}
+
+	filesystem := polyfill.New(&test.BasicMock{})
+	worktree := git.Worktree {
+		Filesystem: filesystem,
+	}
+
+	file, err := ruleFile.Open(&worktree)
+
+	assert.Truef(t, err==nil, "there should be no errors")
+	assert.Truef(t, file!=nil, "there should be a file opened")
+}
+
+func TestGet(t *testing.T) {
+
+
+
+
+	//mockFile.EXPECT().Read(gomock.Any()).Return(0, errors.New("EOF"))
+
```

Comment:
I would remove this function if it doesn't do anything yet. Or at least clean it up.

Reply:
yeah that was a leftover I will update it

Reply:
I temporarily deleted this file